src/sparseml/transformers/export.py

```diff
@@ -297,10 +297,20 @@ def export_transformer_to_onnx(
     )
 
     if sequence_length is None:
+        if hasattr(config, "max_position_embeddings"):
```

Comment:
Could a `config` have both attributes?  If so, is `max_position_embeddings` preferred over `max_seq_len`, as coded here?

Reply:
Good catch. Regarding both keys being simultaneously present- honestly do not know, I would need to go through more example configs. But preferring max_position_embeddings or max_seq_len sounds reasonable to me.